src/tabs/Tabs.tsx

```diff
@@ -112,6 +112,10 @@ function parseTabList(children: ReactNode): Tab[] {
 export type TabsProps = {
   children: ReactNode[];
   className?: string;
+  /**
+   * If specified, the index of the selected tab is controlled by the parent component rather than the internal state.
+   */
+  selectedTabIndex?: number;
```

Comment:
Let's switch this to be `index` as that's the most aria compliant naming: https://github.com/reach/reach-ui/blob/43f450db7bcb25a743121fe31355f2294065a049/packages/tabs/src/reach-tabs.tsx#L82